generate.go

```diff
@@ -72,6 +72,10 @@ func (zp *ZoneParser) generate(l lex) (RR, bool) {
 		s += l.token
 	}
 
+	if (end-start)/step > 65536 {
+		return zp.setParseError("bad range in $GENERATE directive", l)
```

Comment:
Can we merge this with the `end < 0 || start < 0 || end < start` if-statement above?